apps/client/src/features/viewers/lower-thirds/LowerThird.tsx

```diff
@@ -0,0 +1,236 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Message, OntimeEvent, ViewSettings } from 'ontime-types';
+
+import { overrideStylesURL } from '../../../common/api/apiConstants';
+import NavigationMenu from '../../../common/components/navigation-menu/NavigationMenu';
+import { LOWER_THIRD_OPTIONS } from '../../../common/components/view-params-editor/constants';
+import ViewParamsEditor from '../../../common/components/view-params-editor/ViewParamsEditor';
+import { useRuntimeStylesheet } from '../../../common/hooks/useRuntimeStylesheet';
+import { cx } from '../../../common/utils/styleUtils';
+
+import './LowerThird.scss';
+
+enum srcKeys {
+  title = 'title',
+  subtitle = 'subtitle',
+  presenter = 'presenter',
+  lowerMsg = 'lowerMsg',
+}
+
+enum triggerType {
+  event = 'event',
+  manual = 'manual',
+}
+
+type LowerOptions = {
+  width: number;
+  trigger: triggerType;
+  topSrc: srcKeys;
+  bottomSrc: srcKeys;
+  topColour: string;
+  bottomColour: string;
+  topBg: string;
+  bottomBg: string;
+  topSize: string;
+  bottomSize: string;
+  transition: number;
+  delay: number;
+  key: string;
+  lineColour: string;
+  lineHeight: string;
+};
+
+interface LowerProps {
+  eventNow: OntimeEvent | null;
+  viewSettings: ViewSettings;
+  lower: Message;
+}
+
+export default function LowerThird(props: LowerProps) {
+  const { eventNow, lower, viewSettings } = props;
+  const [searchParams] = useSearchParams();
+  const options: LowerOptions = {
+    width: 45,
+    trigger: triggerType.event,
+    topSrc: srcKeys.title,
+    bottomSrc: srcKeys.subtitle,
+    topColour: '000000ff',
+    bottomColour: '000000ff',
+    topBg: '00000000',
+    bottomBg: '00000000',
+    topSize: '65px',
+    bottomSize: '40px',
+    transition: 3,
+    delay: 3,
+    key: 'ffffffff',
+    lineColour: 'ff0000ff',
+    lineHeight: '0.4em',
+  };
+
+  useRuntimeStylesheet(viewSettings?.overrideStyles && overrideStylesURL);
+
+  const [playState, setplayState] = useState<'pre' | 'in' | 'out'>('pre');
+
+  useEffect(() => {
+    document.title = 'ontime - Lower Third';
+  }, []);
+
+  const _width = searchParams.get('width');
+  if (_width) {
+    const width = Number(_width);
+    if (!Number.isNaN(width) && width != options.width) {
```

Comment:
I usually prefer to use `!==` for comparisons as it also ensures both have the same type